proxy/README.md

```diff
@@ -0,0 +1,52 @@
+# OPEA Pipeline Proxy
+
+OPEA Pipeline Proxy is an enhancement of the default Istio proxy with additional features designed specifically for OPEA RAG pipelines.
+
+## Features
+
+- [Guardrails](./deployments/guardrails/README.md)
+
+## Build
```

Comment:
Hi @zhxie, I suggest that this build process can be in a `Makefile` which contains all necessary commands to complete the building.

Reply:
Done, thanks.